src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -1,21 +1,64 @@
-$_event-list-item--max-width: 1126px;
-$_event-list-item--image-height: 166px;
 $_desktop-content-rows: 5;
+$_event-list-item--image-height: 166px;
+$_event-list-item--image-width: 222px;
+$_event-list-item--max-width: 1126px;
+$_stacked-reduce-width: 20px;
 
-.event-list-item {
+@mixin event-list-item-stacked-shadow-box {
+  position: relative;
+  &::before {
+    content: "";
+    background-color: $color__global-white;
+    position: absolute;
+    width: calc(100% - $_stacked-reduce-width);
+    height: 100%;
+    left: 50%;
+    top: 0;
+    transform: translateX(-50%);
+    z-index: -1;
```

Comment:
```suggestion
```

I'd avoid using - values for this, and instead add a higher z-index for -stacked__content